### ISSUE TITLE
online agendas: propose native Linux installs too, recommend Ubuntu

### ISSUE THREAD

**Issue by michaelopdenacker** [bootlin/training-materials#79]
Our current agendas for online sessions, at least the embedded Linux one, says:
"For people interested in our optional practical labs, an installation of VirtualBox and about 30 GB of free disk space." 

Shouldn't instead say?
"For people interested in our optional practical labs, Ubuntu 20.04 in a native installation or in a VirtualBox instance, with about 30 GB of free disk space."

A native installation is much better and it's good to propose the GNU/Linux distro and version that we officially support.
Would you agree with this change?

**Comment by lucaceresoli** [comment]
I agree and I would go even further in _recommending_ a native installation, with a VM only as a fallback.
